packages/fscomponents/src/components/DigitalWalletProvider.tsx

```diff
@@ -44,7 +44,7 @@ export function withDigitalWallet<P extends {}>(
 
   return class DigitalWalletProvider extends React.Component<ResultProps, WithDigitalWalletState> {
     applePay: ApplePay;
-    promises: Promise<any>[] = [];
+    promises: Promise<boolean>[] = [];
```

Comment:
`initApplePay` doesn't return anything so this should be `Promise<void>`.

Reply:
@bweissbart Changes request implemented.